### ISSUE TITLE
Missing Account/Organization Names

### ISSUE THREAD

**Issue by redbassett** [sindresorhus/refined-github#603]
Some account or organization names aren't displayed in the `account/repo` naming format while Refined Github is activated:

![screen shot 2017-07-09 at 18 00 52](https://user-images.githubusercontent.com/1379565/27997955-d43d5cb4-64d0-11e7-8160-a6d45566959a.png)

It looks like the obvious cause of the issue would be the length of repo names, but I haven't confirmed this.  In the image, the second item is `Droidcon-Boston/conference-app-android` and the fifth is `ericwbailey/millennials-to-snake-people`.

(I am using the "Repos you contribute to" box for this screenshot. It is visible because I have overriden RG's hiding of that section. Issue occurs in the normal repo list below as well.)

**Comment by bfred-it** [comment]
Can you look into the dev tools to see what's hiding them?

**Comment by arkon** [comment]
Isn't this box hidden by Refined GitHub?

**Comment by bfred-it** [comment]
Closing this as it doesn't seem to be due to Refined GitHub. Even if I force it to show via userstyle, long repo names are still visible:

<img width="335" alt="screen shot 2017-07-10 at 12 17 22" src="https://user-images.githubusercontent.com/1402241/28002658-bd525c18-6569-11e7-95a1-8356a9a3a1c7.png">

**Comment by redbassett** [comment]
I'm having a hard time finding what is hiding it, however I re-ran my testing.

Originally, this issue went away when I disabled Refined Github, which is why I posted the issue here, but just now, it seems to be persisting? I'm not sure what is up, but despite going away when I turned the extension off before, the issue seems to be not connected to RG. Thanks for reviewing the report.

@arkon: As noted in my original post, I have a userscript to display that box, as I use it regularly.
